src/tests/dotnet-trace/CollectLinuxCommandFunctionalTests.cs

```diff
@@ -0,0 +1,200 @@
+        private static CollectLinuxCommandHandler.CollectLinuxArgs TestArgs(
+            CancellationToken ct = default,
+            string[] providers = null,
+            string clrEventLevel = "",
+            string clrEvents = "",
+            string[] perfEvents = null,
+            string[] profiles = null,
+            FileInfo output = null,
+            TimeSpan duration = default,
+            string name = "",
+            int processId = -1)
+        {
+            return new CollectLinuxCommandHandler.CollectLinuxArgs(ct,
+                                                                   providers ?? Array.Empty<string>(),
+                                                                   clrEventLevel,
+                                                                   clrEvents,
+                                                                   perfEvents ?? Array.Empty<string>(),
+                                                                   profiles ?? Array.Empty<string>(),
+                                                                   output ?? new FileInfo(CommonOptions.DefaultTraceName),
+                                                                   duration,
+                                                                   name,
+                                                                   processId);
+        }
+
+        [Theory]
+        [MemberData(nameof(BasicCases))]
+        public void CollectLinuxCommandProviderConfigurationConsolidation(object testArgs, string[] expectedLines)
+        {
+            MockConsole console = new(200, 30);
+            RunAsync((CollectLinuxCommandHandler.CollectLinuxArgs)testArgs, console);
+            console.AssertSanitizedLinesEqual(null, expectedLines);
+        }
+
+        private static string[] RunAsync(CollectLinuxCommandHandler.CollectLinuxArgs args, MockConsole console)
+        {
+            var handler = new CollectLinuxCommandHandler();
+            handler.Console = console;
+            handler.RecordTraceInvoker = (cmd, len, cb) => 0;
+            int exit = handler.CollectLinux(args);
+            if (exit != 0)
+            {
+                throw new InvalidOperationException($"Collect exited with return code {exit}.");
+            }
+            return console.Lines;
+        }
+
+        public static IEnumerable<object[]> BasicCases()
+        {
+            yield return new object[] {
+                TestArgs(),
+                new string[] {
+                    "No providers, profiles, ClrEvents, or PerfEvents were specified, defaulting to trace profiles 'dotnet-common' + 'cpu-sampling'.",
+                    "",
+                    ProviderHeader,
+                    FormatProvider("Microsoft-Windows-DotNETRuntime","000000100003801D","Informational",4,"--profile"),
+                    "",
+                    LinuxHeader,
+                    LinuxProfile("cpu-sampling"),
+                    ""
+                }
```

Comment:
Shouldn't there be some output about where the test output is being written to and telling the user to press Enter to stop the trace?